examples/golang/append_object.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/smithy-go/ptr"
+
+	"log"
+	"os"
+)
+
+func uploadObject(bucketName, objectName string) {
+	// Open an object:
+	obj, err := os.Open(objectName)
+	if err != nil {
+		log.Println("Open file failed:", err)
+	}
+	defer obj.Close()
+
+	// Create a S3 session:
+	s3session := s3.New(session.Must(session.NewSession(&aws.Config{
+		Region:           aws.String("default"),
```

Comment:
credentials are taken automatically from the aws cli configuration.
you should pass them explicitly here (you can hardcode that in the main functions together with the rest of the vstart specific parameters)

Reply:
Thanks very much for the comments! I will hardcode the credential parameters in the main function.